pkg/tests/integration/test_salt_ufw.py

```diff
@@ -0,0 +1,33 @@
+import logging
+import os.path
+import pathlib
+import subprocess
+
+import pytest
+from pytestskipmarkers.utils import platform
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.skip_on_windows()
+def test_salt_ufw(salt_master, salt_call_cli, install_salt):
+    """
+    Test salt.ufw for Debian/Ubuntu salt-master
+    """
+    log.warning(f"DGM test_salt_ufw install_salt '{install_salt}'")
+
+    if install_salt.distro_id not in ("debian", "ubuntu"):
+        pytest.skip("Only tests Debian / Ubuntu packages")
+
+    # check that the salt_master is running
+    assert salt_master.is_running()
+
+    ufw_master_path = pathlib.Path("/etc/ufw/applications.d/salt-master")
+    assert ufw_master_path.exists()
+
+    etc_ufw_path = pathlib.Path("/etc/ufw/applications.d")
+    str_etc_ufw_path = str(etc_ufw_path)
+    log.warning(f"DGM test_salt_ufw etc ufw contents '{etc_ufw_path}'")
+    ret = salt_call_cli.run("--local", "cmd.run", f"ls -alh {etc_ufw_path}/")
```

Comment:
instead of just checking to make sure the file is there is there a command we could use with `ufw` to validate that its working as well. `ufw allow salt` or some sort of ufw status command that ensures ufw is setup correctly with salt.

Reply:
The bug was to have the file in the correct location, that is why did the minimum to check that it was there, given all the pressure to get code-coverage going.

Will look at adding package test for 'ufw allow salt' being functional, but focusing on the code-coverage